python/instrumentation/openinference-instrumentation-dspy/tests/openinference/instrumentation/dspy/test_instrumentor.py

```diff
@@ -0,0 +1,92 @@
+from typing import Generator
+
+import dspy
+import pytest
+import requests_mock
```

Comment:
Curious, have we thought about using `responses` instead? My impression is that library has more traction.

Reply:
Ah okay yeah let's swap it out in a follow-up